gdpr_assist/models.py

```diff
@@ -419,7 +418,7 @@ def get_anonymisation_tree(cls, prefix="", doprint=False, objs=[]):
             try:
                 field = getattr(cls, flat_field)
 
-                res += "%s|-> %s\n" % (prefix, field.field_name)
+                res += "%s|-> %s\n" % (prefix, flat_field)
```

Comment:
`field_name` is no longer an attribute of the DeferredAttribute object (https://docs.djangoproject.com/en/3.2/_modules/django/db/models/query_utils/)